extension/manifest.json

```diff
@@ -13,12 +13,10 @@
   "permissions": [
     "alarms",
     "experienceSamplingPrivate",
-    "identity",
     "notifications",
     "storage",
     "unlimitedStorage",
-    "https://chrome-experience-sampling.appspot.com/_ah/api/*",
-    "https://www.google.com/*"
+    "https://chrome-experience-sampling.appspot.com/_ah/api/*"
```

Comment:
No, I thought you were using form based POST or GET so was wondering if the permission was needed at all, but it looks like survey submission is XHR based so it is necessary indeed. As you can see I have little memory of this code :)